compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/BasicTooltip.kt

```diff
@@ -382,10 +382,9 @@ object BasicTooltipDefaults {
     const val TooltipDuration = 1500L
 }
 
+@Suppress("EXPECT_ACTUAL_CLASSIFIERS_ARE_IN_BETA_WARNING")
```

Comment:
Our library is full of expect/actuals. It's not supposed to explicitly suppress all of it individually. Is it from AOSP?

Reply:
I think this is only for expect/actual classes. Yes, it's from AOSP.